tc/app/vault/file/upload.help

```diff
@@ -1,4 +1,37 @@
 
-Upload the file to vault server
+tcctl vault file upload --secret-path "PATH" --file "FILE"
 
-    tcctl vault file upload --secret-path "path" --file filename
+tcctl vault file upload --auth-type token \
+                        --token "VAULT_TOKEN" \
+                        --addr "VAULT_ADDR" \
+                        --secret-path "PATH" \
+                        --file "FILE"
+
+tcctl vault file upload --auth-type userpass \
+                        --name "VAULT_NAME" \
```

Comment:
VAULT_NAME -> vault user name or vault login, etc
Only about key description.

Reply:
we agreed to write a variable here, their description is below

`
--token                 "token from vault server"
--name                  "users name on the vault server"
--pass                  "users password on the vault server"
--addr                  "the vault server address"
`